contracts/Erc20Quest.sol

```diff
@@ -65,17 +85,20 @@ contract Erc20Quest is Quest {
         IERC20(rewardToken).safeTransfer(to_, nonClaimableTokens);
     }
 
+    /// @notice Call the QuestFactory contract to get the amount of receipts that have been minted
+    /// @return The amount of receipts that have been minted for the given quest
     function receiptRedeemers() public view returns (uint256) {
         return questFactoryContract.getNumberMinted(questId);
     }
 
+    /// @notice Function that calculates the protocol fee
     function protocolFee() public view returns (uint256) {
-        return receiptRedeemers() * rewardAmountInWeiOrTokenId * questFee / 10_000;
+        return (receiptRedeemers() * rewardAmountInWeiOrTokenId * questFee) / 10_000;
     }
 
+    /// @notice Sends the protocol fee to the protocolFeeRecipient
+    /// @dev Only callable when the quest is active and started
```

Comment:
honestly we should move this to when it's ended. Doesn't make sense to withdraw before

Reply:
Alright, I've changed the modifier so that it uses `onlyAdminWithdrawAfterEnd` rather than `onlyQuestActive`